src/Persistence.LinnApps/Repositories/OverbookAllowedByLogRespository.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Linn.Purchasing.Persistence.LinnApps.Repositories
+{
+    using Linn.Common.Persistence.EntityFramework;
+    using Linn.Common.Proxy.LinnApps;
+    using Linn.Purchasing.Domain.LinnApps.PurchaseOrders;
+
+    public class OverbookAllowedByLogRespository : EntityFrameworkRepository<OverbookAllowedByLog, int>
+    {
+        private readonly ServiceDbContext serviceDbContext;
+        private readonly IDatabaseService databaseService;
+
+        public OverbookAllowedByLogRespository(ServiceDbContext serviceDbContext, IDatabaseService databaseService)
+            : base(serviceDbContext.AllowOverbookLogs)
+        {
+            this.serviceDbContext = serviceDbContext;
+            this.databaseService = databaseService;
+        }
+
+        public override void Add(OverbookAllowedByLog entity)
+        {
+            entity.Id = this.databaseService.GetIdSequence("AN_ARBITRARY_SEQUENCE");
+            entity.LogId = entity.Id;
```

Comment:
why are there two ids?

Reply:
> > Using Richard's SaveToLog contribution to Common that SignalLimitLog is using in my facade service. Just double checking but it doesn't seem that the PL_OVERBOOK_ALLOWED_BY table - any idea where I'm going wrong here?
> 
> do you need to do a TransactionManager commit perhaps?

Aye, it does remind me of that to be honest. I'll go through the SignalLog code to double check this.

Reply:
no you shouldn't have to commit. it's called before the update or add commits anyway

Reply:
> why are there two ids?

This was something I wasn't too sure on (left it out initial commits). The entity.Id is a field within the table whilst the LogId is in the Linn.Common.Domain.LogTable abstract class, wasn't too sure if it needed to be set in the same way that the LogAction is set.